frontend/src/__tests__/cypress/cypress/fixtures/e2e/dataScienceProjects/dataScienceProject.yaml

```diff
@@ -1,4 +1,12 @@
 # testProjectCreation.cy.ts Test Data #
 dsProjectName: "Cypress Test Project 1"
 dsProjectDescription: "Cypress Test project 1 description."
-dsOCProjectName: "cypress-test-project-1"
\ No newline at end of file
+dsOCProjectName: "cypress-test-project-1" 
```

Comment:
I would rename it to `projectName` or `projectResourceName`

Reply:
Resolved in latest commit.